std/datetime/timezone.d

```diff
@@ -781,7 +781,10 @@ public:
 
     @safe unittest
     {
-        assert(LocalTime().dstName !is null);
+        version(CRuntime_Musl)
```

Comment:
A comment here would probably help people reading this in the future.

Reply:
Done, this thread seems to confirm it's not set by default, TooTallNate/node-time#94.